core/capabilities/vault/capability.go

```diff
@@ -58,11 +56,11 @@ func (s *Capability) Execute(ctx context.Context, request capabilities.Capabilit
 		return capabilities.CapabilityResponse{}, errors.New("capability does not support v1 requests")
 	}
 
-	if request.Method != vault.MethodGetSecrets {
+	if request.Method != vaultcommon.MethodGetSecrets {
```

Comment:
This is incorrect IIRC: we should just use the constants that are defined in core (I marked the vaultcommon ones as deprecated yesterday)